CMakeLists.txt

```diff
@@ -63,16 +110,14 @@ set(PHOENIX_LIBRARIES CTRE_Phoenix
                       CTRE_PhoenixCore
 )
 
-# Create a target for each Phoenix library
-foreach(LIBRARY ${PHOENIX_LIBRARIES})
-  add_library(${LIBRARY} STATIC IMPORTED)
-  set(LIBRARY_LOCATION ${CMAKE_CURRENT_SOURCE_DIR}/Phoenix-Linux-SocketCAN-Example/lib/${CTRE_ARCH}/lib${LIBRARY}.a)
-  SET_TARGET_PROPERTIES(${LIBRARY} PROPERTIES IMPORTED_LOCATION ${LIBRARY_LOCATION})
-  install (
-    FILES ${LIBRARY_LOCATION}
-    DESTINATION lib
-  )
-endforeach()
+
+# Download the CTRE libraries
```

Comment:
Are you sure the libs should be tabbed out like this here? This looks wack.

Reply:
yeah that's how joey has it in the original macro on master

Reply:
it's yucky thoughhh 😂 your call, do what you think is best